pages/dr9/sga.rst

```diff
@@ -10,178 +10,866 @@
+(current and former) undergraduate students at `Siena College`_:
+
+- `Alissa Ronca`_
+- `Luis Villa`_
+- `Kevin Napier (University of Michigan)`_
+
+.. _`Data Access`: https://sga.legacysurvey.org
+.. _`SGA web-portal`: https://sga.legacysurvey.org
+.. _`DESI Legacy Imaging Surveys DR9`: ../../dr9
+.. _`DR9`: ../../dr9
+.. _`unWISE`: http://unwise.me
+.. _`Dark Energy Spectroscopic Instrument (DESI)`: http://desi.lbl.gov
+.. _`DESI`: http://desi.lbl.gov
+.. _`HyperLeda`: http://leda.univ-lyon1.fr/
+.. _`HyperLeda extragalactic database`: http://leda.univ-lyon1.fr/
+.. _`Third Reference Catalog of Bright Galaxies (RC3)`: https://vizier.u-strasbg.fr/viz-bin/VizieR?-source=VII/155
+.. _`RC3`: https://vizier.u-strasbg.fr/viz-bin/VizieR?-source=VII/155
+.. _`NASA Extragalactic Database (NED)`: https://ned.ipac.caltech.edu
+.. _`McConnachie (2012)`: https://ui.adsabs.harvard.edu/abs/2012AJ....144....4M/abstract
+.. _`OpenNGC`: https://github.com/mattiaverga/OpenNGC
+.. _`DR8`: ../../dr8
+.. _`this notebook`: https://github.com/legacysurvey/legacypipe/blob/master/doc/nb/lslga-from-gaia.ipynb
+.. _`Principal Catalogue of Galaxies`: https://ui.adsabs.harvard.edu/abs/1989A%26AS...80..299P/abstract
+.. _`DR9 Tractor catalogs`: ../catalogs
+.. _`external catalogs documentation`: ../../external/#sga-large-galaxies
+.. _`DR9 files documentation`: ../files
+.. _`legacypipe`: https://github.com/legacysurvey/legacypipe
+.. _`CCDs`: ../files/#survey-ccds-camera-dr9-fits-gz
+.. _`metrics`: ../files/#other-files
+.. _`DR9 image stacks`: ../files/#image-stacks-region-coadd
+.. _`DR9 bitmasks`: ../bitmasks
+.. _`Tractor catalog`: ../catalogs
+.. _`Tractor catalogs`: ../catalogs
+.. _`PSF documentation`: ../description/#psf
+.. _`astropy`: https://docs.astropy.org/en/stable 
+.. _`photutils`: https://photutils.readthedocs.io/en/stable/isophote.html
+.. _`photutils.isophote.Ellipse.fit_image`: https://photutils.readthedocs.io/en/stable/api/photutils.isophote.Ellipse.html#photutils.isophote.Ellipse.fit_image 
+.. _`photutils.isophote.IsophoteList`: https://photutils.readthedocs.io/en/stable/api/photutils.isophote.IsophoteList.html#photutils.isophote.IsophoteList
+.. _`photutils.isophote.Isophote`: https://photutils.readthedocs.io/en/stable/api/photutils.isophote.Isophote.html#photutils.isophote.Isophote
+.. _`astropy.QTable`: https://docs.astropy.org/en/stable/api/astropy.table.QTable.html#astropy.table.QTable
+.. _`Michele Cappellari's mge.find_galaxy`: https://www-astro.physics.ox.ac.uk/~mxc/software/#mge
+.. _`photutils.aperture`: https://photutils.readthedocs.io/en/stable/aperture.html
+.. _`Observational Astronomy by Birney, Gonzalez, & Oesper`: https://www.cambridge.org/core/books/observational-astronomy/98B4694421AEB3953FE088D19BA0495C
+.. _`DR9 Tractor catalogs`: ../catalogs
+.. _`Scientific Publication Acknowledgement`: ../../acknowledgement/#scientific-publication-acknowledgement
+.. _`SGA acknowledgement`: ../../acknowledgement/#siena-galaxy-atlas
```

Comment:
`acknowledgment`, here, is the preferred U.S. spelling (rather than `acknowledgement`). Normally, I wouldn't care about such pedantry. But, here the usage is in a link. This means the link to:
```
../../acknowledgment/#scientific-publication-acknowledgment
```
is broken. I suggest checking all links using the rendered page at

http://faraday.uwyo.edu/~admyers/scratch/legacysurvey/sga/dr9/sga/

Reply:
Thanks, noted and updated. But note that the "alternative" spelling appears on our home page (`Acknowledgements`), which is what caused me to adopt this spelling without thinking or double-checking the link(s)--
  https://www.legacysurvey.org/